playwright-tests/tests/editor.spec.js

```diff
@@ -276,4 +276,84 @@ test.describe("?page=projects&tab=editor", () => {
+                image: {
+                  url: "https://nearcatalog.xyz/wp-content/uploads/nearcatalog/race-of-sloths.jpg",
+                },
+                backgroundImage: "",
+                tags: {
+                  "developer-support": "",
+                  ecosystem: "",
+                },
+                linktree: {
+                  twitter: "https://twitter.com/race_of_sloths",
+                  github: "https://github.com/race-of-sloths",
+                  telegram: "https://t.me/race_of_sloths",
+                  website: "https://race-of-sloths.com/",
+                },
+              },
+            },
+          },
+          "builddao.testnet": {
+            project: {
+              "saswat_test.testnet_project_race-of-sloths": "",
+            },
+          },
+        },
+      };
+
+      const transactionObj = JSON.parse(
```

Comment:
We already have a test that confirms the transactionObj -- we really just need this test to confirm that the fields are prepopulated

Reply:
Ok, I'll edit this accordingly